de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorManipulator.java

```diff
@@ -151,10 +147,8 @@ public boolean replaceText(SPath path, String text) {
      *
      * @param path
      * @param operations
-     * @param color
      */
-    public void applyTextOperations(SPath path, Operation operations,
-        Color color) {
+    public void applyTextOperations(SPath path, Operation operations) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=240&issues=AWYbBs9UvOWp7Yd63TpV&open=AWYbBs9UvOWp7Yd63TpV)

[ISSUE_KEY]: AWYbBs9UvOWp7Yd63TpV